index.js

```diff
@@ -17,6 +17,7 @@ class Spinnies {
       failColor: 'red',
       spinner: terminalSupportsUnicode() ? dots : dashes,
       disableSpins: false,
+      continuous: false,
```

Comment:
What about changing `continuous` to something like: `keepSpinners`?

Reply:
`keepSpinners` sounds good to me.